packages/winpanda/extra/src/winpanda/core/command.py

```diff
@@ -212,52 +217,57 @@ def execute(self):
         """Execute command."""
         msg_src = self.__class__.__name__
 
-        for pkg_manifest in self.config.inst_storage.get_pkgactive():
+        pkg_manifests = (
+            self.config.inst_storage.get_pkgactive(PackageManifest.load)
+        )
+
+        for pkg_manifest in pkg_manifests:
             pkg_id = pkg_manifest.pkg_id
             LOG.debug(f'{msg_src}: Execute: Manifest: {pkg_manifest.body}')
-            svc_conf = cfp.ConfigParser()
-            svc_conf.read_dict(pkg_manifest.svc_conf)
-            cluster_conf = cfp.ConfigParser()
-            cluster_conf.read_dict(self.config.cluster_conf)
-            svc_manager = WinSvcManagerNSSM(
-                svc_conf=svc_conf, cluster_conf=cluster_conf
-            )
 
-            try:
-                ret_code, stdout, stderr = svc_manager.status()
-            except svcm_exc.ServiceManagerCommandError as e:
-                err_msg = (f'Execute: Get service status (initial):'
-                           f' {pkg_id.pkg_name}: {e}')
-                raise cr_exc.StartCommandError(err_msg)
-            else:
-                LOG.debug(
-                    f'{msg_src}: Execute: Get service status (initial):'
-                    f' {pkg_id.pkg_name}: stdout[{stdout}] stderr[{stderr}]'
+            if pkg_manifest.pkg_svccfg:
+                svc_manager = WinSvcManagerNSSM(
+                    svc_conf=pkg_manifest.pkg_svccfg
                 )
-                svc_status = str(stdout).strip().rstrip('\n')
+            else:
+                svc_manager = None
 
-            if svc_status == SVC_STATUS.STOPPED:
+            if svc_manager:
```

Comment:
With this change, this `if` body can be merged into the previous `if`.

Reply:
Good point. Thanks. Will fix it in the next commit to this PR.